src/InPluginUpsells/resources/js/recurring-donations-settings-tab.js

```diff
@@ -0,0 +1,4 @@
+import {render} from 'react-dom';
+import {RecurringDonationsTab} from './components/RecurringDonationsTab';
+
+render(<RecurringDonationsTab />, document.getElementById('give-in-plugin-upsells'));
```

Comment:
When the recurring add-on is activated, this container will be removed, but the URL will remain the same and this script will be loaded. So we have to check if element exists in DOM before rendering the app.

Reply:
> we could conditionally load these scripts based on whether the recurring add-on is activated or not.

This is what we should do.

Reply:
Agreed. The easiest way to do this is to update the `RecurringDonationsTab::isShowing()` method by adding an additional check if the recurring add-on is installed or not. We can do something like this: `if ( defined('GIVE_RECURRING_VERSION') ) ...`